phase1/gce/DEVEL.md

```diff
@@ -0,0 +1,146 @@
+   instead of the default behavior, which is to install the most recent version.
+
+    ```diff
+       "master")
+    -    kubeadm init --discovery "token://${TOKEN}@" --skip-preflight-checks --api-advertise-addresses "$(get_metadata "k8s-advertise-addresses")"
+    +    kubeadm init --discovery "token://${TOKEN}@" --skip-preflight-checks --api-advertise-addresses "$(get_metadata "k8s-advertise-addresses")" --use-kubernetes-version v1.6.0-alpha
+    ```
+
+5. There are some additional images used during the kubernetes install that are
+   not built as part of kubernetes. Tag recent versions of these images with the
+   same version as your build. kubeadm will attempt to download these images and
+   install them, and it will use the version specified above.
+
+   Replace gcp-project-id with your project id.
+
+
+    ```sh
+    gcloud docker -a
+
+    docker pull gcr.io/google_containers/etcd-amd64:3.0.14-kubeadm
+    docker tag gcr.io/google_containers/etcd-amd64:3.0.14-kubeadm gcr.io/<gcp-project-id>/etcd-amd64:3.0.14-kubeadm
+    docker push gcr.io/<gcp-project-id>/etcd-amd64:3.0.14-kubeadm
+
+    docker pull gcr.io/google-containers/pause-amd64:3.0
+    docker tag gcr.io/google-containers/pause-amd64:3.0 gcr.io/<gcp-project-id>/pause-amd64:3.0
+    docker push gcr.io/<gcp-project-id>/pause-amd64:3.0
+
+    docker pull gcr.io/google-containers/kube-discovery-amd64:1.0
+    docker tag gcr.io/google-containers/kube-discovery-amd64:1.0 gcr.io/<gcp-project-id>/kube-discovery-amd64:1.0
+    docker push gcr.io/<gcp-project-id>/kube-discovery-amd64:1.0
+    ```
+
+6. Create a shell script like this in your kubernetes directory, called
+   `upload.sh`. There is an improvement for this step in flight.
+
+    ```sh
+    #!/bin/bash
+
+    set -o xtrace
+    set -o nounset
+    set -o pipefail
+    set -o errexit
+
+    declare -r KUBE_VERSION="v1.6.0-alpha"
+
+    if [[ -z ${KUBE_REPO_PREFIX:-} ]]; then
+      echo "You must define \$KUBE_REPO_PREFIX. Something like:"
+      echo
+      echo "export KUBE_REPO_PREFIX=gcr.io/<gcp_project_name>"
+      echo
+      exit 1
+    fi
+
+    build_and_push() {
+      local component="$1"
+      bazel run "//build:${component}"
+      docker tag "gcr.io/google-containers/build:${component}" "${KUBE_REPO_PREFIX}/${component}-amd64:${KUBE_VERSION}"
+      docker push "${KUBE_REPO_PREFIX}/${component}-amd64:${KUBE_VERSION}"
+    }
+
+    components=("${@}")
+
+    for component in "${components[@]}"; do
+      build_and_push "${component}"
+    done
+    ```
+
+7. Upload a custom build to a GCS bucket.
+
+    ```sh
+    cd kubernetes
+    gcloud docker -a
+    bash upload.sh kube-apiserver kube-controller-manager kube-proxy kube-scheduler
+    ```
+
+8. Create a cluster
+
+    ```sh
+    go run hack/e2e.go -kubernetes-anywhere-cluster=k-a -kubernetes-anywhere-path ../kubernetes-anywhere -kubernetes-anywhere-phase2-provider=kubeadm -deployment kubernetes-anywhere -v -up
```

Comment:
I was following some other instructions. Not aware of the `kubernetes-anywhere` make targets. Will look into that.